air_quality_monitoring/raqmn/raqmn_data/templates/raqmn_data/plot.html

```diff
@@ -35,8 +35,8 @@
 	{% endfor %}
 	var graph = new Rickshaw.Graph({
```

Comment:
Does this mean that we are now using rickshaw for the plotting? I don't recall the original change for this...